### PR TITLE
[14.0] [MIG] base_jsonify module

### PR DESCRIPTION
**Pull request by chafique-delli**

### CODE REVIEW

**Review by bealdav — APPROVED** [review]

File: base_jsonify/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
 * BEAU Sébastien <sebastien.beau@akretion.com>
 * Raphaël Reverdy <raphael.reverdy@akretion.com>
 * Laurent Mignon <laurent.mignon@acsone.eu>
+* Chafique Delli <chafique.delli@akretion.com>
```
Comment (simahawk) [inline base_jsonify/readme/CONTRIBUTORS.rst]: plain one 2 one migration, not sure this is needed :)

**Review by simahawk — COMMENTED** [review]
LG, would you mind pls giving a review to https://github.com/OCA/server-tools/pull/1894 which would be nice to include into v14 as well?

**Comment by hparfr** [comment]
Hi Chafique, can you please add this one too https://github.com/OCA/server-tools/pull/1912 ?

**Comment by sebastienbeau** [comment]
ping @chafique-delli

**Comment by lmignon** [comment]
Hi @chafique-delli 

Thank you for the migration... As @simahawk I think it would be nice to give a review to #1894  to include it into V14. 
Otherwise LGTM.

**Review by PierrickBrun — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**